### PR TITLE
fix: Remove copy to clipboard own address

### LINKED ISSUE #2961: Remove copy to clipboard own address while clicking on Transfer
When I go to my profile and **click on transfer**, it copies the address to my clipboard.

Is there made intentionally? 
I guess we should not push addresses to user's clipboard without it's own intent

I've sometimes noticed that why have own address in clipboard, now I know where it's coming from 😅

image

h/t @cryptodamsky

### PR DESCRIPTION
**Pull request by petersopko**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #2961
- [x] Same thing was happening for all of these without any reason (or toast informing that copying happened):

- Credit
- Identity
- Transfer
- Transform

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [x] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=<My_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address>)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/shared/format/Identity.vue [hunk 1]
```diff
@@ -114,8 +113,7 @@ export default class Identity extends mixins(InlineMixin) {
   }
 
   get discord(): Address {
-    const discord = this.identity.discord
-    return discord
+    return this.identity.discord
```
Comment (kkukelka) [inline components/shared/format/Identity.vue:116]:
```suggestion
    return this.identity?.discord
```

**Review by kkukelka — APPROVED** [review]
lgtm, gonna commit my own suggestion

**Comment by kkukelka** [comment]
pay 50

**Comment by kkukelka** [comment]
pay 50

**Comment by kkukelka** [comment]
@yangwao payment bot broken?

**Comment by yangwao** [comment]
> @yangwao payment bot broken?

@petersopko did not fill up address :p
but has noted in #2116 that bot should post if did not detect payout address or be more verbose about why the payout wasn't sent, i.e. out of balance, webhook did not hit serverless gw etc

**Comment by petersopko** [comment]
> pay 50

updated address

**Comment by kkukelka** [comment]
pay 50

**Comment by kkukelka** [comment]
pay 50

**Comment by kkukelka** [comment]
pay 50

**Comment by yangwao** [comment]
Ah well that's what I was worried about, @KngZhi seems your regex doesn't parse this address 😛

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
> Execution timeout of 15000ms exceeded.

lol something tell me not work on Sundays

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
> Missing address to be paid at 2970

@KngZhi yeah so did not parsed with this 

target
```
https://beta.kodadot.xyz/transfer/?target=FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y
```

```
// match KSM address, experimental
// const lookupForAddress = /kodadot.xyz\/transfer\/\?target=\w[a-zA-Z0-9]{46,}\b/g;
const lookupForAddress = /kodadot\.xyz\/transfer\?target=\w[a-zA-Z0-9]{46,}\b/g;

// const matchAddress = /\w[a-zA-Z0-9]{46,}\b/g;
const matchAddress = /\w[a-zA-Z0-9]{46,}\b/g;
```

**Comment by KngZhi** [comment]
oh, no~~ dear @yangwao 

the lookupForAddress should be `/kodadot\.xyz\/transfer\/?\?target=\w[a-zA-Z0-9]{46,}\b/g`

please copy that.

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
now webhook did not arrived

pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 114.77 USD/KSM ~ 0.436 $KSM
🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)
🔗 [0x6aa936f7d90a846cb552d69abfc744d63bef926f1f349c020578ff5a1d14f8d5](https://kusama.subscan.io/extrinsic/0x6aa936f7d90a846cb552d69abfc744d63bef926f1f349c020578ff5a1d14f8d5)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**